### DISCUSSION TITLE
Share of Spend vs Share of Effect (QA - Issue)

### DISCUSSION THREAD

**Discussion by TerzoVii** [facebookexperimental/Robyn#382] [Q&A]
Hello everyone and thanks for taking the time to read my case.

Im using the Robyn Algorithm for a while and im my final running test i noticed something that looks off according too many factors (ie total spend for this channel is low according to other channels).

In my case a specific channel (subchannel to be hones t-- NonBrand) has a share of spend vs share of effect from 10% to 33% and avg roi 98.

My model has Rsqr-train 0.87.

May know someone what may be wrong? One other factor that im thinking about is that some other channels have null (0) spent for alot weeks like (40% of the total date frame)
![image](https://user-images.githubusercontent.com/56385779/166508011-1ce18612-0572-45fe-9d93-44c57d089ea5.png)

**Comment by kyletgoldberg** [comment]
Hi, is this a weekly or daily model? Given many of the adstocks are landing at 30% you may want to increase the ranges of the hyperparams so they have more room to try to find the optimal number. I would also take a second look at your data and make sure everything looks correct there.

**Reply by TerzoVii** [reply]
Hi, my data is based on weekly model -- i will give it a try with increase on the range of the hyperparams; 

May my data is the issue because there are alot weeks with 0's (no values) and then a huge spike on the spending. I did a combination of NonBrand one more case and looks like the model improved.

So, im thinking that if we have a long date range with empty values and suddenly a huge increase on spend can affect the output -- what do you think on that?
